cli/vm/cli.go

```diff
@@ -730,6 +741,34 @@ func handleLoadTx(c *cli.Context) error {
 	return nil
 }
 
+func handleLoadDeployed(c *cli.Context) error {
+	err := prepareVM(c, nil)
+	if err != nil {
+		return err
+	}
+	id, err := getContractID(c)
+	if err != nil {
+		return err
+	}
+	bc := getChainFromContext(c.App)
+	h, err := bc.GetContractScriptHash(id) // @fixme: can be improved after #2702 to retrieve historic state of destroyed contract by ID.
```

Comment:
Ah, that's the answer. Can we do this now?

Reply:
Fixed.